query-sql/src/lib.rs

```diff
@@ -268,6 +271,15 @@ impl QueryFragment for Constraint {
                 Ok(())
             },
 
+            &Or { ref constraints } => {
+                out.push_sql("(");
```

Comment:
It's actually `AND ()` we need to worry about — this will be joined into a sequence of `AND`.

I'll add some error checking, probably producing `1` or something.